### PR TITLE
expose versions as top-level tf variables, bump versions to latest

### PR DESCRIPTION
**Pull request by mrparkers**

### CODE REVIEW

File: tf/k8s/variables.tf [hunk 1]
```diff
@@ -26,5 +26,11 @@ variable "enable_jenkins" {
 }
 variable "rode_ui_host" {}
 variable "rode_ui_version" {
-  default = "v0.1.1"
-}
\ No newline at end of file
+  default = "v0.3.0"
```
Comment (alexashley) [inline tf/k8s/variables.tf:29]: Is there value in having a default when it's always overridden by the Terragrunt?

**Review by alexashley — APPROVED** [review]
LGTM, just one question. 

Running locally the Jenkins jobs are red, but it's because the pipeline is trying to push to ECR.

File: tf/k8s/variables.tf (hunk 1 above)
Comment (mrparkers) [reply tf/k8s/variables.tf:29]: yeah, the idea is just to make sure that we have a sane set of variables if you decide to remove those vars from the terragrunt file.  so if I were to update my local `terragrunt.hcl` file to point somewhere else, and I removed these variables, everything would still work

**Review by alexashley — CHANGES_REQUESTED** [review]
Failing after that most recent commit, I think the Rode version needs to be bumped still:

```
 kubectl -n rode-demo logs -f rode-856b99f54d-d4rq9                                                          0 docker-desktop 15:40:23
flag provided but not defined: -port
Usage of /rode:
  -basic-auth-password string
    	when set, basic auth will be enabled for all endpoints, using the provided password. --basic-auth-username must also be set
  -basic-auth-username string
    	when set, basic auth will be enabled for all endpoints, using the provided username. --basic-auth-password must also be set
  -debug
    	when set, debug mode will be enabled
  -elasticsearch-host string
    	the Elasticsearch endpoint used by Grafeas (default "http://elasticsearch-master:9200")
  -elasticsearch-password string
    	password for the Grafeas Elasticsearch instance
  -elasticsearch-refresh string
    	refresh controls when changes made by a request are made visible to search. Options are "true", "false", "wait_for" (default "true")
  -elasticsearch-username string
    	username for the Grafeas Elasticsearch instance
  -grafeas-host string
    	the host to use to connect to grafeas (default "localhost:8080")
  -grpc-port int
    	the port that the rode gRPC API server should listen on (default 50051)
  -http-port int
    	the port that the rode HTTP API server should listen on (default 50052)
  -jwt-issuer string
    	when set, jwt based auth will be enabled for all endpoints. the provided issuer will be used to fetch the discovery document in order to validate received jwts
  -jwt-required-audience aud
    	when set, if jwt based auth is enabled, this audience must be specified within the aud claim of any received jwts
  -opa-host string
    	the host to use to connect to Open Policy Agent (default "http://localhost:8181")
2021/04/14 20:39:39 failed to build config: flag provided but not defined: -port
```

File: tf/k8s/terragrunt.hcl [hunk 2]
```diff
@@ -13,5 +13,6 @@ inputs = {
   enable_nginx    = true
   enable_jenkins  = true
   rode_ui_host    = "rode-ui.localhost"
-  rode_ui_version = "v0.2.0"
+  rode_ui_version = "v0.3.0"
+  rode_version = "v0.3.0"
```
Comment (alexashley) [inline tf/k8s/terragrunt.hcl:17]:
```suggestion
  rode_version    = "v0.5.0"
```

File: tf/k8s/variables.tf (hunk 1 above)
Comment (ChrisSchreiber) [reply tf/k8s/variables.tf:29]: if it is not set in terragrunt it will use the default from the helm chart it its not set here right?

Comment (mrparkers) [reply tf/k8s/variables.tf:29]: the variable here is actually used as the image tag.  I assume we're not going to be publishing helm chart updates every time a release of rode is cut, so we're just setting `image.tag` directly using this variable

**Comment by mrparkers** [comment]
@alexashley yeah I forgot to bump the version in `terragrunt.hcl` as well.  maybe we should make these tf vars required and remove the defaults so it doesn't feel like we're setting them in two places

**Comment by alexashley** [comment]
@mrparkers That sounds reasonable to me.

**Review by alexashley — APPROVED** [review]

**Pull request merged**